www/api/views/nlc_classify.py

```diff
@@ -6,26 +6,9 @@
 
 from classified import Classified
 
+#classifier_id = 'f5bbbcx175-nlc-1012' # Old Classified
 classifier_id = '90e7b4x199-nlc-32429'
 
-def classify_questions(request, sentence):
```

Comment:
This was the old function used when we called the NLC from the client instead of the current method from the server. As such I removed it as it's no longer needed.